core/constants.go

```diff
@@ -167,6 +167,10 @@ const (
 	Fungible ESDTType = iota
 	// NonFungible defines the token type for ESDT non fungible tokens
 	NonFungible
+	// NonFungibleV2 defines the token type for ESDT non fungible tokens
+	NonFungibleV2
+	// Meta defines the token type for ESDT meta tokens
+	Meta
```

Comment:
We have SFT as well. Plus bring here the DynamicNFT, DynamicSFT, DynamicMeta as well.

Reply:
Added.

Reply:
change the order of semiFungible and Meta - maybe add MetaFungible - to have a complete name.

Reply:
+1

Reply:
Done.